scripts/export-framework-applicant-details.py

```diff
@@ -33,11 +34,10 @@
     arguments = docopt(__doc__)
 
     STAGE = arguments['<stage>']
-    API_TOKEN = arguments['<api_token>']
     FRAMEWORK = arguments['<framework_slug>']
     OUTPUT_DIR = arguments['<output_dir>']
 
-    client = DataAPIClient(get_api_endpoint_from_stage(STAGE), API_TOKEN)
+    client = DataAPIClient(get_api_endpoint_from_stage(STAGE), get_auth_token('api', STAGE))
```

Comment:
Can we split removing the api_token from the script into a separate commit? Sorry.

Reply:
Going to remove that from this PR altogether https://github.com/alphagov/digitalmarketplace-scripts/pull/244)

Reply:
That makes sense